prime-router/src/main/kotlin/tokens/Scope.kt

```diff
@@ -0,0 +1,62 @@
+package gov.cdc.prime.router.tokens
```

Comment:
Scopes are crucial to the robustness of the security model.    The design here is to meant to be very simple, but expandable to support any kind of very fine grained or coarse permissions model.  

Note:  there's only one scope really right now, 'report'.